service_crategen/src/cargo.rs

```diff
@@ -0,0 +1,52 @@
+use toml;
+
+use std::collections::BTreeMap;
+
+#[derive(Debug, Default, Clone, Deserialize, Serialize)]
+pub struct Manifest {
```

Comment:
I know! I couldn't find anything to use for this. I figured maybe the `cargo` crate would, but it doesn't. `cargo-edit`, also does not, it works on `toml::Value`. If anything comes up, this should be pretty easy to swap out for something else.